### ISSUE TITLE
GraphQL Enterprise 1.1.2 `Schema.changesets` method should use safe expansion or return empty set

### ISSUE THREAD

**Issue by joelzwarrington** [rmosolgo/graphql-ruby#3972]
The method recently added in 1.1.2 `Schema.changesets` will return an undefined method error when the schema doesn't use `use GraphQL::Enterprise::Changeset::Release`:

```ruby
class Schema < GraphQL::Schema
end

Schema.changesets

# => NoMethodError: undefined method `changesets' for nil:NilClass
# from /Users/joelwarrington/.rvm/gems/ruby-2.7.2/gems/graphql-enterprise-1.1.2/lib/graphql/enterprise/changeset.rb:12:in `changesets'
```

I would expect it to do one of two things when that's the case:

- return `nil`
- or return an empty set

**Comment by rmosolgo** [comment]
Derp, you're so right! I updated it to return an empty set and released v1.1.3. Thanks for reporting this!

**Comment by joelzwarrington** [comment]
❤️  thank you
